Sources/StreamVideo/OpenApi/generated/Models/Count.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+    
+public struct Count: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
what's this one about?

Reply:
Used in `CountrywiseAggregateStats` which is used in `AggregatedStats`.